app/mailers/notification_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class NotificationMailer < ApplicationMailer
+  layout 'notification_email'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.